components/tidb_query_vec_expr/src/types/function.rs

```diff
@@ -274,7 +274,7 @@ pub fn validate_expr_arguments_lte(expr: &Expr, args: usize) -> Result<()> {
 }
 
 thread_local! {
-    pub static VARG_PARAM_BUF: std::cell::RefCell<Vec<usize>> =
+    pub static VARG_PARAM_BUF: std::cell::RefCell<Vec<Option<usize>>> =
```

Comment:
The size of `Option<&T>` is equal to `usize` , documented in https://doc.rust-lang.org/std/mem/fn.size_of.html

Reply:
I prefer to change to `usize`, and then adding a [static assert](https://docs.rs/static_assertions/0.1.1/static_assertions/) for their memory sizes. In this way we maintain readability as well as feel safer..

Reply:
I've changed from Option<&usize> to usize in latest commit